docker-compose-mysql-5-7.yml

```
@@ -3,3 +3,4 @@ version: "3.7"
 services:
   db:
     image: mysql:5.7
+    command: --skip-ssl
```

Maybe the commit message is enough, but it stands out a bit:

```suggestion
    command: --skip-ssl # The image does not support TLS 1.2 which is required recent Java versions
```

---

Did you look at re-writing the config file for Java? I think one can even write a new one, just containing the lines needed to get TLS 1.1 going and provide the path (probably) via `JAVA_OPTS`. 

That solution would be a bit more advanced, but at the other hand also more intention revealing.

---

It might be possible to override the `java.security` file according to [this page](https://dzone.com/articles/how-override-java-security), it depends on the value of `security.overridePropertiesFile` in the file. I guess I'll have to find the file path in the container and `cat` it in actions in order to get get that answered?

---

We could commit a custom file in https://github.com/twingly/ecco/tree/master/docker and point to that one

---

> I guess I'll have to find the file path in the container and cat it in actions in order to get get that answered?

Found it and overrides are allowed. However, I'm not sure about the property name here. [I tried](https://github.com/twingly/ecco/runs/2664075647?check_suite_focus=true#step:4:18) `-Djruby.security.properties`, but it wasn't recognized.

---

It's probably `-Jjava.security.properties`

---

I suppose one could set `JAVA_OPTS`, to not nest it into the `JRUBY_OPTS`, I think both are taken into account, in that case it's the usual `-Djava.security.properties`

---

Aha, I ran with `-Djava` first, but in `JRUBY_OPTS`. I'll try the two other alternatives then 👍 

https://github.com/twingly/ecco/runs/2663977191?check_suite_focus=true#step:4:18

---

Cool, `-Djava` in `JAVA_OPTS` works!

---

4c3fb0f